browser/models.py

```diff
@@ -506,3 +510,41 @@ def _history_user(self):
     @_history_user.setter
     def _history_user(self, value):
         self.changed_by = value
+
+
+class Position(YearMixin, CuratedMixin, LastUpdatedMixin):
+    history = HistoricalRecords()
+    person = models.ForeignKey('Person')
+    subject = models.CharField(max_length=255, blank=True)
+    CORPORATION_MEMBER = 'CM'
+    TRUSTEE = 'TR'
+    FRIDAY_EVENING_LECTURER = 'FEL'
+    role_choices = ((CORPORATION_MEMBER, 'Corporation Member'), (TRUSTEE, 'Trustee'),
+                    (FRIDAY_EVENING_LECTURER, 'Friday Evening Lecturer'))
+    role = models.CharField(max_length=255, blank=False, choices=role_choices, default=CORPORATION_MEMBER)
```

Comment:
A word of caution: `blank=False` only applies to forms. If you want the field nullable in the database, make sure you add `null=True` here. Same goes for the rest of the fields.

If you do add `null=True`, squash all the new migrations into one to allow this change specific migrations to be undone.

Reply:
Also, is `max_length=255` actually necessary for `role` when there are only three possible values with maximum string length being 3?

Reply:
I did not want the field to be nullable in the back-end. So, I have used ```blank=False``` to validate in the form itself.
The max_length is 255 so that in the future we can add roles of varied lengths without making changes in the models.

Reply:
Since there are keys it is highly unlikely that we will have keys that are 255 in the future. 3 is probably enough even if we add a lot more options (which is unlikely).

Reply:
this is still set to 255

Reply:
Changed the length of role to 3.